concise-mid-tag.cddl

```diff
@@ -131,9 +131,15 @@ tagged-int-type = #6.551(int)
 ueid-type = bytes .size 33
 tagged-ueid-type = #6.550(ueid-type)
 
+;
+; mkey type definitions
+;
 $measured-element-type-choice /= tagged-oid-type
 $measured-element-type-choice /= tagged-uuid-type
+; The $measured-element-type-choice uint is intended for time series mval data where lower uint
+; values indicate older mval data while higher uint values indicate newer mval data; 0 is an asumed epoch.
```

Comment:
It is expected that mval be used in an increasing set of values, i.e. higher values indicate newer mval data; 0 is an assumed epoch.

Reply:
maybe should say "unsigned integer" instead of integer. Otherwise OK.

Reply:
I am OK with unsigned integer, I will suggest and inline edit sothat you can commit my suggestion:

`It is expected that mkey as unsigned integer is used for mval in an increasing set of values, i.e. higher values of mkey indicate newer mval data; 0 is an assumed epoch.`

Reply:
ok